server/scsynth/SC_CoreAudio.cpp

```diff
@@ -1598,6 +1606,9 @@ bool SC_CoreAudioDriver::DriverStart() {
     if (mWorld->mVerbosity >= 1) {
         scprintf("->SC_CoreAudioDriver::DriverStart\n");
     }
+
+    auto appIOProcFunc = mSafetyClipThreshold < INFINITY ? appIOProc<true> : appIOProc<false>;
```

Comment:
i think you should also disable clipping if the threshold is negative. it would rhyme with the 'negative = off' behavior that pops up in other places in SC. and also, the clipping behavior will not work if the input is negative; if i'm following the definition of sc_clip2 right, the output will just sit at the negative value most of the time, which could lead to loud pops as well.